cleanlab/token_classification/filter.py

```diff
@@ -1,22 +1,24 @@
 import numpy as np
 from cleanlab.filter import find_label_issues as find_label_issues_main
-from typing import Optional, Union, Tuple
+from typing import List, Tuple
 
 
 def find_label_issues(
-    labels: list, pred_probs: list, return_indices_ranked_by: str = "self_confidence"
-) -> list:
+    labels: list,
+    pred_probs: list,
+    return_indices_ranked_by: str = "self_confidence",
+) -> List[Tuple[int, int]]:
     """Returns issues identified by cleanlab
 
     This is a function to find the label issues for token classification datasets
 
     Parameters
     ----------
-    labels: list
+    labels:
```

Comment:
@anishathalye Is it preferable like this vs. the old way: `labels: list` ?

Reply:
Yes, we'll generate the types based on Python annotations, so we don't have duplicates here.